src/test/java/org/folio/rest/impl/UsersAPIIT.java

```diff
@@ -145,4 +165,23 @@ void facetsLimit0() {
   void facetsLimit1() {
     facets(1);
   }
+
+  @Test
+  void delete() {
+    String id1 = "114b14e8-422d-429a-b89e-b55c439b3b21";
+    String id2 = "22531038-08b8-4c71-a72c-9d48f63f3682";
+    String id3 = "13836e0e-4331-4386-a8be-2345eece6de3";
+    postUser(id1, id1);
+    postUser(id2, id2);
+    postUser(id3, id3);
+    given().
+    when().
+      param("query", "username == \"1*\"").
+      delete("/users").
+    then().
+      statusCode(204);
+    userExists(id2);
+    userDoesntExist(id1);
+    userDoesntExist(id3);
```

Comment:
Personally, I'd use some whitespace for differentiate between the arrangement, the act and the checks.

```suggestion
    String id1 = "114b14e8-422d-429a-b89e-b55c439b3b21";
    String id2 = "22531038-08b8-4c71-a72c-9d48f63f3682";
    String id3 = "13836e0e-4331-4386-a8be-2345eece6de3";
    
    postUser(id1, id1);
    postUser(id2, id2);
    postUser(id3, id3);
    
    given().
    when().
      param("query", "username == \"1*\"").
      delete("/users").
    then().
      statusCode(204);
      
    userExists(id2);
    userDoesntExist(id1);
    userDoesntExist(id3);
```

Reply:
Empty line inserted before the tests.